packages/demo-nextjs/middleware.js

```diff
@@ -0,0 +1,28 @@
+// middleware.js
+
+import { NextResponse } from 'next/server';
+import sdkInstance from '@/lib/convertSdk';
```

Comment:
@JosephSamirL I cannot find `lib/convertSdk` anywhere, I assume that's where the ConvertSDk is being initialized with our SDK key. Most probably the `lib` folder is ignored in git (_there's a reason for that_). I think you should rename `lib/convertSdk` to something like `convert/init`

Reply:
Changed to be under file convert because there were two files ignored convertSdk and datastore